src/foam/nanos/notification/email/services.jrl

```diff
@@ -1,5 +1,12 @@
-p({"class":"foam.nanos.boot.NSpec", "name":"emailTemplateDAO",                 "lazy":false,  "serve":true,  "serviceScript":"return new foam.dao.PMDAO(x, new foam.dao.PipelinePMDAO(x, new foam.dao.SequenceNumberDAO(new foam.dao.java.JDAO(x, foam.nanos.notification.email.EmailTemplate.getOwnClassInfo(), \"emailTemplates\"))));","client":"{\"of\":\"foam.nanos.notification.email.EmailTemplate\"}" })
+p({"class":"foam.nanos.boot.NSpec", "name":"localEmailTemplateDAO",                 "lazy":false,  "serve":false,  "serviceScript":"return new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.notification.email.EmailTemplate.getOwnClassInfo()).setAuthenticate(false).setPm(true).setPipelinePm(true).setSeqNo(true).setJournalType(foam.dao.JournalType.SINGLE_JOURNAL).setJournalName(\"emailTemplates\").build();"})
+
+p({"class":"foam.nanos.boot.NSpec", "name":"emailTemplateDAO",                 "lazy":false,  "serve":true,  "serviceScript":"return new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.notification.email.EmailTemplate.getOwnClassInfo()).setInnerDAO((foam.dao.DAO) x.get(\"localEmailTemplateDAO\")).build();","client":"{\"of\":\"foam.nanos.notification.email.EmailTemplate\"}" })
```

Comment:
this and and pretty much all other services are set `authorize:false` (not set at all but it is the default). As I mentioned above I think the default should be true or explicitly set to true. I'd imagine only local services would not require authorization